testsuite/features/secondary/min_config_state_channel_subscriptions.feature

```diff
@@ -60,7 +60,7 @@ Feature: State Configuration channels
     And I check "statechannel-cbox"
     And I check "statechannel2-cbox"
     When I click on "Save Changes"
-    Then I should see a "Edit Channel Ranks" text
+    And I wait until I see "Edit Channel Ranks" text
     And I should see a "My State Channel (statechannel)" link
```

Comment:
Maybe use `Then` here

Reply:
No, we use `When` in that kind of step. I also think `Then` would be better, but that's what was agreed. Main reason, the step contains an action: "waiting"